src/networking.c

```diff
@@ -1477,6 +1480,12 @@ void unlinkClient(client *c) {
         c->flags &= ~CLIENT_UNBLOCKED;
     }
 
+    if (c->user) {
+        ln = listSearchKey(c->user->clients, c);
```

Comment:
it's better to add an aux node pointer like `client_list_node` to accelerate, avoid iterate the whole list.

Reply:
good idea!